### PR TITLE
Upgrade for Android API 29 , using gradle 6, react-native 0.61+, test…

### PR DESCRIPTION
**Pull request by ralic**
…ed on windows10

### CODE REVIEW

**Comment by Overtorment** [comment]
Is this package maintained? Will this PR be merged?

File: TcpServer.js [hunk 1]
```diff
@@ -31,16 +27,16 @@ function TcpServer(connectionListener: (socket: Socket) => void) {
   this._socket = new Socket();
 
   // $FlowFixMe: suppressing this error flow doesn't like EventEmitter
-  this._socket.on('connect', function() {
+  this._socket.on('connect', function () {
```
Comment (phillbaker) [inline TcpServer.js:30]: Can you separate out these formatting changes into a separate PR?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,567 @@
+{
```
Comment (phillbaker) [inline package-lock.json:1]: This file shouldn't be committed to this repo.

**Comment by phillbaker** [comment]
Published `react-native-tcp@4.0.0` which should work with react-native >= 0.59 and androidx.

**Comment by dac245** [comment]
I don't see the change from the build.gradle in the new version from this, mainly the changes where compile is changed to implementation.
